addon/components/polaris-banner.js

```diff
@@ -183,7 +183,9 @@ export default Component.extend({
   didRender() {
     this._super(...arguments);
 
-    let hasContentWrapper = this.$().find('div.Polaris-Banner__Content').length;
+    let hasContentWrapper = this.element.querySelectorAll(
```

Comment:
Can this be `querySelector` instead of `querySelectorAll`?

Reply:
Ah you're right. It's only here to check if it's present in the DOM so it can just return 1 node (in fact I'm not sure if this would ever return more than 1 item anyway)